check/test.go

```diff
@@ -232,7 +236,22 @@ func (t testItem) evaluate(s string) *testOutput {
 	}
 
 	result.flagFound = match
-	glog.V(3).Info(fmt.Sprintf("found %v", result.flagFound))
+	var isExist string
+	if !result.flagFound{
+		isExist = " not"
+	}
+	if t.auditUsed != "auditCommand"{
+		if t.auditUsed != "auditConfig"{
+			glog.V(3).Info(fmt.Sprintf("Env '%v'%s exist", t.Env, isExist))
+		} else {
+			glog.V(3).Info(fmt.Sprintf("Path '%v'%s exist", t.Path, isExist))
+		}
+	} else {
+		glog.V(3).Info(fmt.Sprintf("Flag '%v'%s exist", t.Flag, isExist))
+	}
```

Comment:
I think this would be nicer as a switch something like this: 

```suggestion
    switch t.auditUsed {
        case "auditCommand": 
		glog.V(3).Info(fmt.Sprintf("Flag '%v' %s exist", t.Flag, isExist))
        case "auditConfig":
		glog.V(3).Info(fmt.Sprintf("Path '%v' %s exist", t.Flag, isExist))
        ...
    }
```

As an aside, generally speaking if you're using if tests try to test on the positive sense so it's easier to follow the logic: `if t.auditUsed == "auditCommand"` is easier to hold in your head than `if t.auditUsed != "auditCommand"`. But I think a switch will be neater here anyway